pkg/utils/rand.go

```diff
@@ -231,16 +231,16 @@ func RandOutputOnAddressWithAmount(outputType iotago.OutputType, address iotago.
 	return iotaOutput
 }
 
-func RandPubKey() ed25519.PublicKey {
-	return ed25519.PublicKey(RandBytes(32))
+func RandBlockIssuerKey() iotago.BlockIssuerKey {
+	return iotago.BlockIssuerKeyEd25519FromPublicKey(ed25519.PublicKey(RandBytes(32)))
 }
 
-func RandPubKeys() []ed25519.PublicKey {
+func RandBlockIssuerKeys() iotago.BlockIssuerKeys {
 	length := RandomIntn(10)
```

Comment:
Should that maybe be `length := RandomIntn(9) +1 ` so we always have at least 1 IssuerKey?

Reply:
Or is it good to also test the "no block issuer keys available" case? :D